packages/zowe-explorer/src/globals.ts

```diff
@@ -56,7 +56,7 @@ export const DS_TEXT_FILE_CONTEXT = "textFile";
 export const DS_FAV_TEXT_FILE_CONTEXT = "textFile_fav";
 export const DS_BINARY_FILE_CONTEXT = "binaryFile";
 export const DS_MIGRATED_FILE_CONTEXT = "migr";
-export const DS_IMPERATIVE_ERROR_CONTEXT = "impError";
+export const DS_FILE_ERROR_CONTEXT = "impError";
```

Comment:
Do we want to keep `impError` as the context or update to reflect rename to `file`?

Reply:
🤦 I forgot about that one 😋